diff --git a/CMakeLists.txt b/CMakeLists.txt
index 14a48cb..cfda2fd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -49,6 +49,8 @@ ExternalProject_Add(OpenEXR
     CMAKE_ARGS
         -DCMAKE_INSTALL_PREFIX:PATH=<INSTALL_DIR>
         -DBUILD_SHARED_LIBS=OFF
+        -DCMAKE_C_COMPILER=/usr/bin/gcc-11
+        -DCMAKE_CXX_COMPILER=/usr/bin/g++-11
     DEPENDS ${CHAIN}
 )
 set(CHAIN OpenEXR)
@@ -100,6 +102,8 @@ ExternalProject_Add(OpenColorIO
         -DBUILD_SHARED_LIBS=ON
         -DOCIO_BUILD_STATIC=OFF
         -DCMAKE_CXX_STANDARD=17
+        -DCMAKE_C_COMPILER=/usr/bin/gcc-11
+        -DCMAKE_CXX_COMPILER=/usr/bin/g++-11
     DEPENDS ${CHAIN}
 )
 set(CHAIN OpenColorIO)
